pubspec.yaml

```diff
@@ -40,15 +40,20 @@ dependencies:
   device_preview: 0.4.1+1
   intl: ^0.16.1
   audioplayers: 0.15.1
+  flutter_native_image: 0.0.5
   flutter_audio_recorder: ^0.5.5
-  hive: 1.4.1+1
-  hive_flutter: 0.3.0+1
+  hive:
+  hive_flutter:
```

Comment:
Why do we need the fork?

Reply:
Conflict with Flutter intl on dev

Reply:
Let's fix version to 1.17/stable and this issue is resolved 
![image](https://user-images.githubusercontent.com/16854239/81580630-f8bd4400-93ad-11ea-963a-140dc53c00e8.png)